ntbs-service/Pages/LegacyNotifications/Index.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@page "/LegacyNotifications/{LegacyNotificationId}"
+
+@using ntbs_service.Helpers
+@model ntbs_service.Pages.LegacyNotifications.Index
+
+@{
+    var classSuffix = CssClassHelpers.GetClassSuffixForNotificationStatus(Model.NotificationBanner.NotificationStatus);
+    var notificationStatusTriangleCssClass = $"notification-banner-triangle notification-banner-triangle{classSuffix}";
+}
+
+<nhs-width-container container-width="Standard">
+
+    <nhs-grid-row>
+        <nhs-grid-column grid-column-width="Full">
+            <div class="notification-banner-container">
+                <partial name="../Shared/_NotificationBanner.cshtml" model="@Model.NotificationBanner" />
+                <div class="@notificationStatusTriangleCssClass"> </div>
+            </div>
+        </nhs-grid-column>
+    </nhs-grid-row>
+    
+    @if (Model.LegacyImportResult == null)
+    {
+        <div class="ondemand-migration-description-container">
+            <p>You must import this notification to view the full details of the record.</p>
+            <br/>
+            <p>Import may take a few moments.</p>
+            <br/>
+            <p>Do you wish to proceed?</p>
+            <br/>
+            <form method="post">
+                <button class="nhsuk-button">Import</button>
+            </form>
+        </div>
+    }
+    else
+    {
+        <nhs-error-summary title-text="This notification cannot be imported" title-id="error-summary-title"
+                           role="alert" tabindex="-1" data-module="error-summary">
+            @foreach (var (key, validationErrors) in Model.LegacyImportResult.ValidationErrors)
+            {
+                <li><strong>@validationErrors.Count validation errors found for notification with Id = @key</strong></li>
+                @foreach (var item in validationErrors)
+                {
+                    <li>@item</li>
+                }
+            }
+        </nhs-error-summary>
+        
+        <h5>What next?</h5>
+        <span>Please contact ntbs@phe.gov.uk quoting reference ID: {@Model.RequestId} </span>
```

Comment:
Can I suggest that we also ask the user to quote the legacy id of the notification? I know it's a deviation from ticket's definition, but after thinking about it, this seems much more relevant and useful!